sematic/BUILD

```diff
@@ -34,7 +34,9 @@ sematic_py_lib(
     srcs = ["user_settings.py"],
     deps = [
         ":config_dir",
-        requirement("pyyaml"),
+    ],
+    pip_deps = [
```

Comment:
Had to change all the `BUILD` files to use `pip_deps` instead of `requirement` with the normal `deps` so we could more easily handle the interpreter-specific dependency of pip deps distinctly from interpreter-specific dependency of sematic deps.